opencti-platform/opencti-front/src/private/components/common/drawer/DrawerOpenCTI.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FunctionComponent } from 'react';
+import Drawer from '@mui/material/Drawer';
+import makeStyles from '@mui/styles/makeStyles';
+import IconButton from '@mui/material/IconButton';
+import { Close } from '@mui/icons-material';
+import Typography from '@mui/material/Typography';
+import { Theme } from '../../../../components/Theme';
+import useAuth from '../../../../utils/hooks/useAuth';
+
+const useStyles = makeStyles<Theme>((theme) => {
+  const {
+    bannerSettings: { bannerHeightNumber },
+  } = useAuth();
+  return ({
+    drawerPaper: {
+      minHeight: '100vh',
+      width: '50%',
+      position: 'fixed',
+      overflow: 'auto',
+      transition: theme.transitions.create('width', {
+        easing: theme.transitions.easing.sharp, duration: theme.transitions.duration.enteringScreen,
+      }),
+      paddingTop: `${bannerHeightNumber}px`,
+    },
+    header: {
+      backgroundColor: theme.palette.background.nav,
+      padding: '10px 0',
+      display: 'inline-flex',
+      alignItems: 'center',
+    },
+    container: {
+      padding: '10px 20px 20px 20px',
+    },
+  });
+});
+
+interface DrawerOpenCTIProps {
+  title: string
+  open: boolean
+  setOpen: (value: boolean) => void
+  children: React.ReactNode
+}
+
+const DrawerOpenCTI: FunctionComponent<DrawerOpenCTIProps> = ({
```

Comment:
How costly is it to add the <Fab> inside the drawer ?

Reply:
As discussed with Romuald today, we will add the Fab with a variant inside the Drawer. The only specific case is TriggerCreation that will not have the new custom drawer.

Reply:
Is not possible for now :
Style in ReportCreation ->   createButton: {
    position: 'fixed',
    bottom: 30,
    right: 30,
  },

Style in NotifierCreation ->   createButton: {
    position: 'fixed',
    bottom: 30,
    right: 230,
  },


Because of the right bar
![image](https://github.com/OpenCTI-Platform/opencti/assets/53513584/87d54b34-41c3-429c-8fde-f9bed26c446e)